decidim-core/lib/decidim/assets/tailwind/tailwind.config.js.erb

```diff
@@ -47,7 +47,10 @@ module.exports = {
         3: "#E1E5EF",
         4: "#242424",
         5: "#F6F8FA",
-        6: "#D3D5D9"
+        6: "#D3D5D9",
```

Comment:
This one is for @julietapasetti 

Can we review these new grays to use the ones that we already have or the neutral palette?